sources/packages/web/src/components/generic/StatusChip.vue

```diff
@@ -1,18 +1,14 @@
 <template>
-  <v-chip :text-color="textColor" :class="chipClass" variant="outlined">
-    <v-icon start icon="fa:fa fa-circle" :color="iconColor" size="12" />
-    {{ label ?? status }}
+  <v-chip :color="chipColor" variant="outlined" :class="chipBackground"
+    ><v-icon :color="iconColor" :icon="icon" size="18" />
```

Comment:
Is the size="18" the same as what we have in figma ?

Reply:
all the places where I have added size for the icon were big than the Figma, for eg.. 
1. below is the icon with default size
![image](https://user-images.githubusercontent.com/77353155/182709768-c11fc1bd-6ca7-4846-969e-2f4ea9ac76bd.png)
below is the figma,
![image](https://user-images.githubusercontent.com/77353155/182709985-8336880f-9998-4383-bd2f-cc4a83b7b28a.png)